tests/unit/prediction/ocr/test_packager_code.py

```diff
@@ -11,6 +11,7 @@
         ("Sustainable palm oil RSPO-5068502 ", ["RSPO-5068502"]),
         ("RSPO-50685022", []),
         ("QRSPO-2404885", []),
+        ("FR-012-345", ["FR-012-345"]),
```

Comment:
you should add some text where it does not match (eg FR-A22-322) and also put the code inside a sentence to verify it's working fine.

Reply:
also fr-333-444 should not match (upper cased)